### PR TITLE
Refactoring of CID docs - add CFN for CUR replication,  set manual deployment as deprecated and add new CFN as experimental

### PR DESCRIPTION
**Pull request by iakov-aws**
*Issue #, if available:*

*Description of changes:*
* add new cfn for cur replication
* make manual as deprecated (no deletion yet)
* add experimental CFN

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md [hunk 1]
```diff
@@ -80,17 +126,17 @@ cid-cmd deploy
 
 {{% /expand%}}
 
-### Option 2: CloudFormation Deployment (30 Mins)
```
Comment (aaronedell) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:83]: I'm wondering why we have this option at all anymore?

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md [hunk 2]
```diff
@@ -2,12 +2,58 @@
 title: "CID CUDOS and the KPI Dashboards"
 date: 2021-08-30T11:16:08-04:00
 chapter: false
-weight: 2
+weight: 4
 pre: "<b>2. </b>"
 ---
 The Cost Intelligence Dashboard (CID), CUDOS, and the KPI dashboard are all strongly recommended to be deployed together. These three dashboards are based on the Cost & Usage report and will help you address a wide range of FinOps use cases. The following steps will help you deploy all three dashboards into your account. If you just want to deploy one or two of these dashboards instead, please review the manual deployment section.
 
-## Express Deployment (15 minutes)
```
Comment (aaronedell) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:10]: I'm not sure we should say this option is experimental. We should probably say it is "preferred" and explain what it is.

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md [hunk 3]
```diff
@@ -2,12 +2,58 @@
...
+## (New) Experimental CloudFormation deployment
+{{%expand "Click here to continue" %}}
+
+1. Login into your Data Collection Account
```
Comment (aaronedell) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:13]: We need a one sentence explainer of what this option is.

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md [hunk 4]
```diff
@@ -2,12 +2,58 @@
...
+ 1) Check if QuckSight has permissions to access Athena, S3 bucket with CUR and S3 bucket with athena results. 
+ 2) Check the status of datasets. If they are being updated
+ 3) Check if CUR data arrived to S3 bucket (If you just created CUR you will need to wait 24 hours before the first data arrives).
+
+
+{{% /expand%}}
+
+
+## Option 1: Express Deployment (15 minutes)
 The following is the **recommended method** for deploying the CUR-based Cloud Intelligence Dashboards (CID, CUDOS, and KPI). For the CloudFormation method or the manual method, please see options 2 or 3.
```
Comment (aaronedell) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:57]: We need to probably remove this and switch it with the one above? Each option should have a sentence explaining it.

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1a_cur_setup.md [hunk 5]
```diff
@@ -0,0 +1,510 @@
+---
+title: "Prerequisites: Cost And Usage Report Setup"
+date: 2022-05-20T11:16:08-04:00
+chapter: false
+weight: 1
+pre: "<b>1a. </b>"
+---
+
```
Comment (aaronedell) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1a_cur_setup.md:8]: I think this whole page needs a lot of clarity work. I think it's okay to merge for now but I'd like to then quickly do a PR with some clean up.

**Review by aaronedell — COMMENTED** [review]
Some concerns mostly around the way things are worded. Comments in line.

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md (hunk 1 above)
Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:83]: I want to keep the new as experimental for a couple of weeks as we grow in number of deployments we will delete manual and depecate old cfn

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md (hunk 2 above)
Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:10]: not yet. soon.

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md (hunk 4 above)
Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:57]: later, yes

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1a_cur_setup.md (hunk 5 above)
Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1a_cur_setup.md:8]: ok

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md (hunk 2 above)
Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:10]: will add a wording explaining more

Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:10]: done please check

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1a_cur_setup.md [hunk 6]
```diff
@@ -0,0 +1,510 @@
...
+   
+2. Click the **Launch CloudFormation button** below to open the **pre-populated stack template** in your CloudFormation console and select **Next**.
+
+	- [Launch CloudFormation Template](https://console.aws.amazon.com/cloudformation/home#/stacks/new?&templateURL=https://aws-managed-cost-intelligence-dashboards.s3.amazonaws.com/cfn/cur-aggregation.yaml&stackName=CID-CUR-Destination)
+	
+![Images/multi-account/cf_dash_2.png](/Cost/200_Cloud_Intelligence/Images//multi-account/cf_dash_launch_2.png?classes=lab_picture_small)
+
+4. Enter your **Destination** Account Id. 
+   
+![Images/multi-account/cfn_dash_param_dst_dedicated_1.png](/Cost/200_Cloud_Intelligence/Images/multi-account/cfn_dash_param_dst_dedicated_1.png?classes=lab_picture_small)
```
Comment (yprikhodko) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1a_cur_setup.md:107]: We agreed to rephrase param name to 'Enter destination AWS account id' and make a new screenshot so it uses the same terminology

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1a_cur_setup.md [hunk 7]
```diff
@@ -0,0 +1,510 @@
...
+![Images/cf_dash_9.png](/Cost/200_Cloud_Intelligence/Images/cf_dash_9.png?classes=lab_picture_small)
+
+10. You will see the stack will start in **CREATE_IN_PROGRESS**.
+**NOTE:** This step can take 5-15mins
+    ------------ | -------------
+
+11. Once complete, the stack will show **CREATE_COMPLETE**.
+
+
+### Create CUR in Source Account using CloudFormation
```
Comment (yprikhodko) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1a_cur_setup.md:131]: seems like this section can't be rendered. Have a look in local server mode.

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1a_cur_setup.md [hunk 8]
```diff
@@ -0,0 +1,510 @@
...
+Sync existing objects from CUR S3 bucket to S3 bucket in Data Collection Account.
+
+	aws s3 sync s3://{curBucketName} s3://{DataCollectionAccountBucketName} --acl bucket-owner-full-control
+
+After performing this step in each management (payer) account S3 bucket in Data Collection Account will contain CUR data from all payer accounts under respective prefixes.
+{{% /expand%}}
+
+
+## Configure Athena
+You will need to do this in Data Collection Account, and only if you choose Manual or Automated multi-account option of CUR Deployment. The latest version of CID CloudFormation does not require this step.
```
Comment (yprikhodko) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1a_cur_setup.md:439]: The latest version of CID CloudFormation does not require this step - this should be removed. As a customer when I read this how do I know what exactly you mean by The latest version of CID CloudFormation

Comment (yprikhodko) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1a_cur_setup.md:439]: I'd suggest to re-phrase. 'If you used Option 2. Single account deployment skip this step and proceed with Glue Crawler set up'

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1a_cur_setup.md [hunk 9]
```diff
@@ -0,0 +1,510 @@
...
+- Additional report details: Include **Resource IDs**
+- Time Granularity: **Hourly**
+- Report Versioning: **Overwrite existing report**
+- Report data integration for: **Amazon Athena**
+- Compression type: **Parquet**
+
+If you have a Cost & Usage Report that meets this criteria you can use it, if not, you can create a new CUR and request a [backfill](https://docs.aws.amazon.com/cur/latest/userguide/troubleshooting.html#backfill-data).
+
+
+## CUR Deployment
```
Comment (yprikhodko) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1a_cur_setup.md:28]: should we mark top level steps with numbers like: 1/3 Create CUR, 2/3 Configure Athena, 3/3 Create Glue Crawler?

**Review by yprikhodko — CHANGES_REQUESTED** [review]

**Review by yprikhodko — APPROVED** [review]

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1a_cur_setup.md [hunk 10]
```diff
@@ -0,0 +1,510 @@
...
+{{%expand "Click here to continue with Cloud Formation Deployment" %}}
+
+If you have __one or several management (payer) accounts__ we recommend to install Dashboads in a dedicated AWS Account. In this case you will need to create Cost & Usage Reports to export data to S3 in each management (payer) account and then configure an S3 replication to the dedicated account. The replicaiton data volume is relatively small.
+
+![Images/multi-account/Architecture1.png](/Cost/200_Cloud_Intelligence/Images/multi-account/Architecture1.png?classes=lab_picture_small)
+
+If you have just one management (payer) account, a dedicated account for dashboards is still a recommended option.
+
+Another frequent use case is __multi linked account__ setup. When AWS Customer has a number of AWS Accounts but no access to management (payer) account. In this case it is possible to configure CUR in each account and set up a replication to one account that will be used for dashboards. Thus the replication architecture is similar to the schema with multi-payer described on the schema above. An only difference is that in this case a local CUR must be activated in the destination account.
```
[the remark this answers is not in the document]
Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1a_cur_setup.md]: done

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1a_cur_setup.md (hunk 6 above)
Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1a_cur_setup.md]: done

**Pull request merged**